config/21.1/plugins.conf

```diff
@@ -1,78 +1,78 @@
 #ORIGIN						IGNORE
-benchmarks/iperf				arm,arm64
```

Comment:
changes are a bit intrusive here... I don't see a clear agenda?

Reply:
Here I just want to enable as many packages as possible for the arm. Most of the packages here are already supported in FreeBSD 12.1 ports, and I have also successfully built them.